src/components/AppHeader/AppHeader.jsx

```diff
@@ -1,6 +1,10 @@
-// import Logo from "@/assets/spa-logo.png";
-// import "./AppHeader.scss";
-// const AppHeader = () => (
-// what goes in here? there is no content in the AppHeader component
-// );
-// export default AppHeader;
+import Logo from "@/assets/spa-logo.png";
```

Comment:
I believe that te correct file is hotel-logo.png rather than spa-logo.png, as we can see in the last step of the issue: "render the **hotel's logo** in an <img>".